schemas/iso19139/src/main/plugin/iso19139/loc/dut/schematron-rules-iso.xml

```diff
@@ -202,8 +202,8 @@
   <alert.M23>
     <div>Either 'aggregateDataSetName' or 'aggregateDataSetIdentifier' must be documented.</div>
   </alert.M23>
-  <report.M23>aggregateDataSetName or aggregateDataSetIdentifier documented.</report.M23>
-<report.M23-dup>An associated resource is declared multiple time with same association and initiative type.</report.M23-dup>
+  <report.M23>De naam of de titel van de verwante dataset is gegeven.</report.M23>
+  <report.M23-dup>An associated resource is declared multiple time with same association and initiative type.</report.M23-dup>
   <alert.M24>
     <div>language not present.</div>
```

Comment:
Well... I could start translating this whole file but I'm not sure all strings are actually in use. Is it necessary, @fxprunayre ?

Reply:
:) No we can check later if more translations needs to be back ported here. I had conflicts while merging on the first one, so that's why I added it here.